pkg/reconciler/revision/queueing_test.go

```diff
@@ -111,6 +111,30 @@ func testRevision() *v1.Revision {
 	return rev
 }
 
+func testRevisionForMultipleContainer() *v1.Revision {
```

Comment:
can we call regular `testRevision` here and just annotate podspec?
remove duplicate boilerplate.